src/lib/operators/sort.cpp

```diff
@@ -88,15 +90,34 @@ class Sort::SortImplMaterializeOutput {
         auto column_it = columns_out.begin();
         auto chunk_it = output_columns_by_chunk.begin();
         auto chunk_offset_out = 0u;
+
+        auto column_ptr_and_accessor_by_chunk_id =
```

Comment:
Maybe it would be better if "sort" would not materialize the output. That's not part of this PR though.